lib/output/utils.js

```diff
@@ -36,11 +47,21 @@ function initCtorRegistry(globalObject) {
     ctorRegistry[`%${intrinsic}%`] = globalObject[intrinsic];
   }
 
-  // TODO: Also capture `%AsyncIteratorPrototype%`
+  ctorRegistry["%Object.prototype%"] = globalObject.Object.prototype;
   ctorRegistry["%IteratorPrototype%"] = Object.getPrototypeOf(
     Object.getPrototypeOf(new ctorRegistry["%Array%"]()[Symbol.iterator]())
   );
```

Comment:
Should I add fallbacks for `"%IteratorPrototype%"`, `"%Object.prototype%"`, and/or `"%Array%"` too?

Reply:
Nah. `eval` is a sufficiently weird case to treat differently.